Integrations/integration-email-sender.yml

```diff
@@ -0,0 +1,349 @@
+description: Send emails implemented in Python with embedded image support
+detaileddescription: |-
+  ## SMTP sender
+  Send emails including support for rich emails with HTML and embedded files.
+  - Most fields are optional and we allow empty subject and empty body
+  - If both body and htmlBody are provided, we will create an alternative envelope
+  - Preconfigured template attachments can be supported by using [data URLs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs)
+
+  Implemented in Python to support extensions.
+configuration:
+- display: Mail server hostname or IP address
+  name: host
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: SMTP Port
+  name: port
+  defaultvalue: "25"
+  type: 0
+  required: true
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: false
+- display: Sender address
+  name: from
+  defaultvalue: noreply@demisto.com
+  type: 0
+  required: true
+- display: Use TLS for connection
+  name: tls
+  defaultvalue: "false"
+  type: 8
+  required: false
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    from email.mime.audio import MIMEAudio
+    from email.mime.base import MIMEBase
+    from email.mime.image import MIMEImage
+    from email.mime.multipart import MIMEMultipart
+    from email.mime.text import MIMEText
+    from smtplib import SMTP
+    import base64
+    import mimetypes
+    from email import encoders
+    import re
+    import random
+    import string
+
+
+    # Following methods raise exceptions so no need to check for return codes
+    SERVER = SMTP(demisto.getParam('host'), int(demisto.getParam('port')))
```

Comment:
You are not handling proxy params
Need to remove proxy env vars if proxy is disabled

Reply:
Proxy is for HTTP - not related to SMTP (at least we never handled it)